src/openapi/peridio-device-openapi.yaml

```diff
@@ -272,9 +274,83 @@ paths:
                           type: string
                         signatures:
                           $ref: "peridio-admin-openapi.yaml#/components/schemas/array-of-signatures"
+  /tunnels/{tunnel_prn}/configure:
+    post:
+      operationId: tunnel-configure
+      summary: configure tunnel
+      description: |
+        Takes parameters to help negotiate configuring Wireguard tunnel between the proxy and device
```

Comment:
Configures a previously requested remote access tunnel with device networking data.

If successful, the configured tunnel will be `open`.